### ISSUE TITLE
Create a Sidebar component

### ISSUE THREAD

**Issue by antw** [quintel/refresh#18]
Create a Sidebar and related components (groups, sidebar items, etc). Each group should be an accordion (Reach UI?) to ensure accessibility.

- [x] Add React component
- [x] Support nested pages
- [x] Add support for mini bar chart
